includes/API/AdPartner/AdGroupApi.php

```diff
@@ -71,29 +88,65 @@ public function create( $campaign_data ) {
 		 */
 		$payload = array(
 			'data' => array(
-				'bid_type'                     => 'CPC',
-				'bid_value'                    => Helper::amount_to_microcurrency( 1 ),
+				'bid_type'                     => null,
 				'campaign_id'                  => $campaign_id,
 				'configured_status'            => 'ACTIVE',
 				'goal_type'                    => 'DAILY_SPEND',
 				'goal_value'                   => Helper::amount_to_microcurrency( (float) $daily_budget ),
-				'name'                         => Helper::get_store_name( 'ad_group' ),
+				'name'                         => Helper::get_store_name( 'ad_group_' . $targeting_type ),
```

Comment:
Similar question here, what if $targeting_type is `''`?

Reply:
Nothing will fail but the ad group name will end with weird `_`. However, this is be fixed now. Thanks for the note.